supervisor/docker/addon.py

```diff
@@ -153,6 +163,15 @@ def cgroups_rules(self) -> Optional[List[str]]:
         if self.addon.with_usb:
             rules.update(self.sys_hardware.policy.get_cgroups_rules(PolicyGroup.USB))
 
+        # Full Access
+        if not self.addon.protected and self.addon.with_full_access:
+            if rules:
+                _LOGGER.warning(
```

Comment:
If `not self.addon.protected` there is no need to warn? they get a huge banner in the UI when it's disabled

Reply:
The warning is about a missconfig